styles/variables-gallium.less

```diff
@@ -174,13 +174,11 @@
 
 // Item
 // ---------------------------------------
-@agate-item-font-size: 30px;
-@agate-item-font-weight: 300;
-@agate-item-label-font-size: @agate-base-font-size;
-@agate-item-height: 24px;
-@agate-item-side-outset: 12px;
-@agate-item-side-outset: 12px;
-@agate-item-height: 60px;
+@agate-item-font-size: 40px;
+@agate-item-font-weight: 500;
+@agate-item-label-font-size: 30px;
+@agate-item-side-outset: 32px;
```

Comment:
This seems a bit large to me. Was this pulled from the specs?

Reply:
Yea 32px is a bit large but it's from here:
<img width="387" alt="itempadding" src="https://user-images.githubusercontent.com/8940009/65540226-cebb1e00-debf-11e9-9f4a-c8ed9aa0957f.png">

Reply:
I see. That seems to be consistent for items other than the menu items.